frontend/src/react/Instructions/Instructions.js

```diff
@@ -26,18 +22,62 @@ export default class Instructions extends Component {
         this.setState({[e.target.name]:e.target.value});
     };
 
+    /**
+     * Send an exam to the API and redirect to /newexam/:examId/exercices if success
+     * @return undefined
+     */
+    addExamAndRedirect = async () => {
+        const { title,
+                reminder,
+                instruction} = this.state;
+        const examData = {
+            title,
+            reminder,
+            instruction
+        };
+
+        await this.props.createExam(examData);
+        if (this.props.exams.exam._id) {
+            this.setState({
+                idRedirect: this.props.exams.exam._id,
+                redirectExercices: true})
+        }
+    };
+
+    triggerActive = (e) => {
```

Comment:
I'd say this function as well as the inactive one should be put inside an helper file of some sort. I'm pretty confident we will have to use that again later!